docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -2407,6 +2407,8 @@ This section contains the following parameters:
   * hostname_levenshtein_distance - just like nearest_hostname, but it compares hostname in a levenshtein distance manner.
   * first_or_random - selects the first ZooKeeper node, if it's not available then randomly selects one of remaining ZooKeeper nodes.
   * round_robin - selects the first ZooKeeper node, if reconnection happens selects the next.
+- `compressed_protocol` — If set to true, enables compression in keeper protocol.
```

Comment:
Regarding the name, I took the same from issue description. https://github.com/ClickHouse/ClickHouse/issues/49507 On second thought, I like `use_compression` better, @alesapin : what do you feel about `use_compression` instead of `compressed_protocol`?

Reply:
Hard to decide, to me both sounds ok, let's have `use_compression` if @antonio2368 likes it.

Reply:
addressed it